mozregression/main.py

```diff
@@ -286,10 +289,39 @@ def preference(prefs_files, prefs_args):
     return prefs()
 
 
+def check_mozregression_version():
+    url = "https://pypi.python.org/pypi/mozregression/json"
+
+    LOG.info("Checking for availability of the"
```

Comment:
Hmm, I think it will just annoy users to see that each time they run the command - I think you can remove this line